web/views/page_view.ex

```diff
@@ -1,11 +1,16 @@
 defmodule DevWizard.PageView do
   use DevWizard.Web, :view
+  use Timex
 
-  def review_state(issue) do
-    case issue.review_state do
-      :not_signed_off -> "Needs Review!"
-      :signed_off     -> "Signed Off"
-      _               -> "?"
+  def days_since(date) do
```

Comment:
Good call! This is something we can definitely add later. And there's an opportunity here to open-source a bunch of rails-esque helpers for formatting Timex dates.